src/middlewared/middlewared/plugins/cluster_linux/ctdb_shared_volume.py

```diff
@@ -89,11 +89,12 @@ async def create(self, job):
             options = {'args': (CTDB_VOL_NAME,)}
             await self.middleware.call('gluster.method.run', volume.start, options)
 
-        # try to mount it locally
-        mount_job = await self.middleware.call(
-            'gluster.fuse.mount', {'name': CTDB_VOL_NAME, 'raise': True}
-        )
-        await mount_job.wait(raise_error=True)
+        # try to mount it locally and send a request
+        # to all the other peers in the TSP to also
+        # FUSE mount it
+        data = {'event': 'VOLUME_START', 'name': CTDB_VOL_NAME, 'forward': True}
+        j = await self.middleware.call('gluster.localevents.send', data)
```

Comment:
Its pretty minor, but `job` instead of `j` probably make the code just a little bit more readable :P